Sources/WeatherSensorService/Public/WeatherSensorReadingType+CustomStringConvertible.swift

```diff
@@ -26,8 +26,8 @@ extension DateFormatter {
 }
 
 extension Double {
-    public var toStringTwoDecimalPlaces: String { String(format: "%.2f", self) }
-    public var toStringFourDecimalPlaces: String { String(format: "%.4f", self) }
+    fileprivate var toStringTwoDecimalPlaces: String { String(format: "%.2f", self) }
+    fileprivate var toStringFourDecimalPlaces: String { String(format: "%.4f", self) }
```

Comment:
I had made this public originally, probably thinking that way the candidate could use it, but now I think it should not be public. However, `internal` might be the better access level, that way the rest of the library can use it internally. `internal` is the default, so to make it internal just remove `fileprivate`. BUT - make sure the library is sill using this extension, and if not, just go ahead and remove it.